### PR TITLE
feat(config): extends config yaml with core config and delay mode

### PR DESCRIPTION
**Pull request by tiagolobocastro**
This allows configuration of the core pin per node.

### CODE REVIEW

**Review by niladrih — APPROVED** [review]

File: io-engine/src/core/env.rs [hunk 1]
```diff
@@ -251,6 +251,18 @@ pub struct MayastorCliArgs {
     /// be set to 1.
     #[clap(long = "enable-lvm", env = "ENABLE_LVM")]
     pub lvm: bool,
+    /// Reactors sleeps 1ms before each poll.
```
Comment (dsavitskiy) [inline io-engine/src/core/env.rs]: "Reactor sleeps" (plural vs singular)

**Review by dsavitskiy — APPROVED** [review]

File: io-engine/src/core/env.rs (hunk 1 above)
Comment (tiagolobocastro) [reply io-engine/src/core/env.rs]: Modified to "Reactors sleep"

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**